spec/support/shared_controller_contexts.rb

```diff
@@ -1,3 +1,11 @@
+shared_context 'an unauthorized access to a resource' do
+  it { is_expected.to respond_with(302) }
+  it { is_expected.to redirect_to(root_path) }
+  it do
+    is_expected.to set_flash[:alert].to(I18n.t('item_master.access_denied'))
```

Comment:
I just checked and noticed that there is not localization value for the `access_denied` key, I suggest adding in. :smile: